gulp/styles.js

```diff
@@ -41,7 +41,7 @@ module.exports = {
 function onStylesDone() {
   notifier.notify({ title: 'Gulp', message: 'Styles Task Finished!' });
   // Manually inject css to ensure compiled css will reflect
-  if ( isWatching && browserSync.initialized ) {
+  if ( browserSync.initialized ) {
```

Comment:
I'm not sure why `isWatching` was here. Just `browserSync.initialized` should be enough?